next_generation/Dockerfile.aws

```diff
@@ -6,8 +6,8 @@ RUN mkdir -p ./pint_server
 COPY pint_server/. ./pint_server
 
 
-COPY opt/. /opt
-COPY usr/. /usr
+COPY .staging/opt/. /opt
+COPY .staging/usr/. /usr
```

Comment:
these two are not needed as the Dockerfile is used inside the .staging dir

Reply:
Updated